robottelo/datafactory.py

```diff
@@ -179,9 +179,9 @@ def valid_emails_list():
             gen_string('alphanumeric'),
             gen_string('alphanumeric'),
         ),
-        u'"():;"@example.com',
         u'!#$%&*+-/=?^`{|}~@example.com',
     ]
+    return [email.replace('"', '').replace('`', '') for email in email_list]
```

Comment:
1. genstring here used only with `alpha`, `numeric` and `alphanumeric` parameters, hence it generates only [a-zA-Z0-9], no unicode or special characters. Double quotes and grace accent present only in the lines ``` u'"():;"@example.com'``` and ```u'!#$%&*+-/=?^`{|}~@example.com'```. So it would be enough to edit that lines.

2. But. It is valid characters according to RFC5322 [0] [1] and I doubt about removing them. Shell (or CLI) problem it is a shell problem, the fact that bash requires escaping some chars should not affect proper email validation and API/UI tests.

[0] https://en.wikipedia.org/wiki/Email_address#Local-part
[1] https://tools.ietf.org/html/rfc5322